src/main/java/uk/nhs/hee/tis/trainee/sync/facade/PlacementEnricherFacade.java

```diff
@@ -72,10 +78,12 @@ public class PlacementEnricherFacade {
 
   PlacementEnricherFacade(PostSyncService postService,
       TrustSyncService trustService, SiteSyncService siteService,
+      GradeSyncService gradeServie,
```

Comment:
Just for consistency, could we call this gradeService

Reply:
Oh missing typo. Thanks for the good spot.